docker-compose.yml

```diff
@@ -8,9 +8,9 @@ services:
     command: mysqld --datadir=/tmp/mysqldata --slow_query_log=1 --log_output=TABLE --log-queries-not-using-indexes --event-scheduler=ON
     environment: &mysql-default-environment
       MYSQL_ROOT_PASSWORD: toor
-      MYSQL_DATABASE: kolide
-      MYSQL_USER: kolide
-      MYSQL_PASSWORD: kolide
+      MYSQL_DATABASE: fleet
```

Comment:
@zwass would these need to change too, to adjust for the db name change?



- https://github.com/fleetdm/fleet/blob/8f048dbce3d1ab95f2fdb17d0214e860051e5352/docs/2-Deployment/2-Configuration.md
- https://github.com/fleetdm/fleet/blob/af96e52a002df59de1bd55ba028245fb28b2eefc/tools/app/development.yml
- https://github.com/fleetdm/fleet/blob/c6b285c56e046717bd0b5ac3c352d7fd463dbb0a/tools/minikube.yaml
- https://github.com/fleetdm/fleet/blob/49e71e4ed694eb9be4cc1ca68c6e27f893aaffe3/.circleci/config.yml
- https://github.com/fleetdm/fleet/blob/7d299ca6f7caf74fc01e2052b671da2c3db0f80b/Makefile
- https://github.com/fleetdm/fleet/blob/cf9146eea20ccf5a1314fcea184a9a242f6a9f12/docs/2-Deployment/4-Example-deployment-scenarios.md

(^ cmd+f+"kolide")


The above were the only places I could find by doing a GitHub code search for "kolide mysql" in the repo-- went through all the files and didn't see any others

Reply:
@mikermcneil none of those are directly relevant to this PR, but I will follow up to update and remove some of them.